sw/host/opentitanlib/src/backend/mod.rs

```diff
@@ -47,8 +47,8 @@ pub struct BackendOpts {
     #[structopt(flatten)]
     ti50emulator_opts: ti50emulator::Ti50EmulatorOpts,
 
-    #[structopt(long, help = "Configuration files")]
-    conf: Option<PathBuf>,
+    #[structopt(long, number_of_values(1), help = "Configuration files")]
```

Comment:
Just came across this line with @jwnrt recently. This doesn't make much sense to me? `number_of_values(1)` means exactly one value is expected, how is this different from just an `Option`?